### PR TITLE
Fix AnalyzeAPI with 'explain' parameter

### PR DESCRIPTION
**Pull request by thePanz**
The ElasticSearch AnalyzeAPI has an "explain" parameter to gather further details about the performed analysis on a text content.
Unfortunately the ElasticSearch response is different when such parameter is set or not set.
See https://www.elastic.co/guide/en/elasticsearch/reference/5.2/_explain_analyze.html
and https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-analyze.html.

The format "explain = true" is consistent in the 5.x major release.

### CODE REVIEW

**Comment by Tobion** [comment]
👍 would be nice if you could add a note to the changelog file

**Comment by thePanz** [comment]
@Tobion : Changelog added

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file based on the
 
 ### Improvements
 
+- added support for the "explain" flag of AnalyzeAPI
```
Comment (ruflin) [inline CHANGELOG.md]: @thePanz Could you also add the PR number behind it? We missed it in the recent changelog entries but I want to add it again as it makes reading changelogs easier as you can jump to the PR directly.

Comment (thePanz) [inline CHANGELOG.md]: @ruflin PR number and link added. Does it work OK for you?

**Comment by ruflin** [comment]
@thePanz LGTM. Thanks

**Pull request merged**